### PR TITLE
docs: update dependabot workflow

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
### Related Issue 

Closes: #959 

#### Changes made 👷🏻‍♂️

- add field for a commit message

```yml
commit-message:
      prefix: "chore(deps)"
```

- Add a field for the label `dependencies`, which I believe is the default, just in case you want to change it in the future.

```yml
labels:
      - dependencies
```

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 version: 2
 updates:
   - package-ecosystem: "npm"
-    directory: "/" 
+    labels:
+      - dependencies
+    directory: "/"  # refers to the location of package.json
```
Comment (tamalCodes) [inline .github/dependabot.yml:6]:
```suggestion
    directory: "/" 
```

We can remove the extra comment.

**Comment by Anmol-Baranwal** [comment]
@tamalCodes 
Also, do you have any issues or features that you think I can work on? I want to focus on level 3 PRs or maybe level 2 PRs, but not level 1, as those are the most basic ones and don't really teach me much.

**Comment by tamalCodes** [comment]
@Anmol-Baranwal , are you comfortable with working with the frontend then ?

**Review by tamalCodes — APPROVED** [review]
### Amazing work 🔥
**The changes look good to me and will be merged soon.**

Do follow [Tamal](https://github.com/tamalCodes) for more Opensource fun projects and don't forget to drop a star so that you get updated about our latest [releases](https://github.com/tamalCodes/Milan/releases) (we will tag you and mention your work) and also a [shoutout](https://www.linkedin.com/feed/update/urn:li:activity:7073574804519354368/) on social media (LinkedIn and Twitter) !

Happy Opensource 🚀.

**Comment by Anmol-Baranwal** [comment]
> @Anmol-Baranwal , are you comfortable with working with the frontend then ?

Yeah, pretty much.

**Pull request merged**